site/src/pages/DeploymentSettingsPage/LicensesSettingsPage/LicensesSettingsPageView.tsx

```diff
@@ -151,6 +157,17 @@ const LicensesSettingsPageView: FC<Props> = ({
 						}))}
 					/>
 				)}
+
+				{licenses && licenses.length > 0 && managedAgentFeature && (
+					<ManagedAgentsConsumption
+						usage={managedAgentFeature.actual || 0}
+						included={managedAgentFeature.soft_limit || 0}
+						limit={managedAgentFeature.limit || 0}
+						startDate={managedAgentLimitStarts || ""}
+						endDate={managedAgentLimitExpires || ""}
+						enabled={managedAgentFeatureEnabled}
```

Comment:
...and then we don't need to do this defaulting here. we should do some validation inside of this component and return `null` or an `<ErrorAlert>` if the `feature` passed in is incorrect.